README.md

```diff
@@ -26,8 +26,8 @@ The library relies on the flutter_web_auth package to allow the Authorization Co
 AndroidManifest.xml
 
 ```xml
-<activity android:name="com.linusu.flutter_web_auth.CallbackActivity" android:exported="true">
```

Comment:
A few lines above this, you ought want to change the string to `flutter_web_auth_2` again. In fact, an overall search for `flutter_web_auth` in the entire project might be appropriate.